scripts/format_notebooks.py

```diff
@@ -247,14 +299,20 @@ def preprocess(self, nb, resources):
         "-d",
         "--default",
         action="store_true",
-        help="Perform default formatting, equivalent to -wcnksr",
+        help="Perform default formatting, equivalent to -wcnksrv",
     )
     parser.add_argument(
         "-r",
         "--run_cloud",
         action="store_true",
         help="Add or update cells that support running this notebook via cloud services",
     )
+    parser.add_argument(
+        "-v",
+        "--check_version",
```

Comment:
Oh, one more thing; this name here is a bit peculiar, because it sounds more like a validation than something active (that is, it is a bit confusing with `--check`). Maybe it could be flipped to:

```suggestion
        "--version_checks",
```

or 

```suggestion
        "--version_validation",
```